library/src/main/java/com/getstream/sdk/chat/utils/CaptureController.java

```diff
@@ -0,0 +1,75 @@
+package com.getstream.sdk.chat.utils;
+
+import android.annotation.SuppressLint;
+import android.content.ContentValues;
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Environment;
+import android.provider.MediaStore;
+
+import androidx.annotation.Nullable;
+
+import com.getstream.sdk.chat.R;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class CaptureController {
+
+    private static Uri imageUri, videoUri;
```

Comment:
It's hard to understand why these fields are here. I assume that methods of the class must be called in a particular order, but then it's hard to understand what order exactly. And `getCaptureFile` is public, so developer might call it first and there will be an issue.

If it's possible, it's better to make static method as isolated as possible, without side effects and states (like static fields)